File: app/src/main/java/ru/willdes/nginxplus/nginxplus.java
package ru.willdes.nginxplus;

import android.provider.BaseColumns;

public final class nginxplus implements BaseColumns {
    public static final String COLUMN_ID = "_id";
    public static final String COLUMN_IDSRV = "idsrv";
    public final static String TABLE_NAME = "connections";
    public final static String TABLE_UPSTREAMS = "upstreams";
    public final static String TABLE_SERVERS = "servers";
    public final static String COLUMN_NAME = "name";
    public final static String COLUMN_ADDRESS = "address";
    public final static String COLUMN_USER = "user";
    public final static String COLUMN_PASSWD = "password";
    public final static String COLUMN_PORT = "port";
    public final static String COLUMN_IDCONN = "id_conn";
    public final static String COLUMN_SERVER = "server";
    public final static String COLUMN_STATE = "state";
    public final static String COLUMN_ACTIVE = "active";
    public final static String COLUMN_REQUESTS = "requests";
    public final static String COLUMN_REQPSEC = "rps";
    public final static String COLUMN_IDUPSTR = "id_upstream";
}

File: app/src/main/java/ru/willdes/nginxplus/nginxplus.kt
package ru.willdes.nginxplus

import android.provider.BaseColumns

object nginxplus : BaseColumns {
    const val COLUMN_ID = "_id"
    const val COLUMN_IDSRV = "idsrv"
    const val TABLE_NAME = "connections"
    const val TABLE_UPSTREAMS = "upstreams"
    const val TABLE_SERVERS = "servers"
    const val COLUMN_NAME = "name"
    const val COLUMN_ADDRESS = "address"
    const val COLUMN_USER = "user"
    const val COLUMN_PASSWD = "password"
    const val COLUMN_PORT = "port"
    const val COLUMN_IDCONN = "id_conn"
    const val COLUMN_SERVER = "server"
    const val COLUMN_STATE = "state"
    const val COLUMN_ACTIVE = "active"
    const val COLUMN_REQUESTS = "requests"
    const val COLUMN_REQPSEC = "rps"
    const val COLUMN_IDUPSTR = "id_upstream"
}